package.json

```diff
@@ -67,8 +67,10 @@
     "eslint-plugin-react-hooks": "^2.5.1",
     "graphql": "^14.6.0",
     "husky": "^4.2.3",
+    "i": "^0.3.6",
```

Comment:
That’s accidental I assume? Same for the dependency on `npm`? 😅

Reply:
Amateur hour 🤦‍♂ cheers @kitten